frontend/packages/integration-tests-cypress/support/a11y.ts

```diff
@@ -0,0 +1,50 @@
+import 'cypress-jest-adapter';
+import 'cypress-axe';
+import { Result } from 'axe-core';
+
+declare global {
+  namespace Cypress {
+    interface Chainable<Subject> {
+      logA11yViolations(violations: Result[], target: string): Chainable<Element>;
+      testA11y(target: string): Chainable<Element>;
+    }
+  }
+}
+
+Cypress.Commands.add('logA11yViolations', (violations: Result[], target: string) => {
+  // pluck specific keys to keep the table readable
+  const violationData = violations.map(({ id, impact, description, nodes }) => ({
+    id,
+    impact,
+    description,
+    nodes: nodes.length,
+  }));
+  window.Cypress.numberA11yViolations += violations.length;
```

Comment:
Should we be modifying the `Cypress` object? I'd think we'd want to avoid mutating that since it's not our API and store this somewhere else.

Reply:
Ok, how about `window.a11y.numberViolations` ?